src/components/Download/Download.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useEffect } from "react";
+import "./Download.css";
+import { Table, NavLink, Input, Row, Col } from "reactstrap";
+import { IconContext } from "react-icons";
+import { FaDownload } from "react-icons/fa";
+
+export default () => {
+  const [Platform, setPlatform] = useState("");
+
+  useEffect(() => {
+    const os = window.navigator.platform;
+    setPlatform(os);
+  }, []);
+
+  const download = async () => {
+    const url =
+      "https://api.github.com/repos/MovingBlocks/TerasologyLauncher/releases/latest";
+    const response = await fetch(url);
+    const data = await response.json();
+    let downloadUrl;
+
+    if (Platform == "Win32") {
+      downloadUrl = data.assets && data.assets[3].browser_download_url;
+      window.location.href = downloadUrl;
+    } else if (Platform == "Linux x86_64") {
+      downloadUrl = data.assets && data.assets[0].browser_download_url;
+      window.location.href = downloadUrl;
+    } else if (Platform == "MacIntel") {
+      downloadUrl = data.assets && data.assets[1].browser_download_url;
+      window.location.href = downloadUrl;
+    }
```

Comment:
@majordwarf  
 @ryuk156 
I was wondering if anyone opens it their android device or any device other than these 3 , their must be a default value to which it must fallback.
So we should have a `else` block here to handle that behaviour .

?

Reply:
You're right will think about this else part and will make changes accordingly